khelo/lib/ui/flow/team/detail/components/team_detail_member_content.dart

```diff
@@ -29,59 +27,14 @@ class TeamDetailMemberContent extends ConsumerWidget {
       );
     }
 
-    return ListView(
-      padding: const EdgeInsets.symmetric(horizontal: 16),
-      children: [
-        const SizedBox(
-          height: 24,
-        ),
-        for (final member in state.team?.players ?? []) ...[
-          _memberCell(context, member),
-          const SizedBox(
-            height: 16,
-          ),
-        ],
-        const SizedBox(
-          height: 34,
-        ),
-      ],
-    );
-  }
-
-  Widget _memberCell(BuildContext context, UserModel user) {
-    return Row(
-      children: [
-        ImageAvatar(
-          initial: user.nameInitial,
-          imageUrl: user.profile_img_url,
-          size: 50,
-        ),
-        const SizedBox(
-          width: 16,
-        ),
-        Column(
-          crossAxisAlignment: CrossAxisAlignment.start,
-          children: [
-            Text(
-              user.name ?? context.l10n.common_anonymous_title,
-              style: AppTextStyle.subtitle1
-                  .copyWith(color: context.colorScheme.textPrimary),
-            ),
-            Text(
-              user.player_role != null
-                  ? user.player_role!.getString(context)
-                  : "Not Specified",
-              style: AppTextStyle.subtitle2
-                  .copyWith(color: context.colorScheme.textSecondary),
-            ),
-            Text(
-              user.location ?? "Not Specified",
-              style: AppTextStyle.subtitle2
-                  .copyWith(color: context.colorScheme.textSecondary),
-            )
-          ],
-        )
-      ],
+    return ListView.builder(
+      padding: context.mediaQueryPadding +
+          const EdgeInsets.symmetric(horizontal: 16),
+      itemCount: state.team!.players!.length,
+      itemBuilder: (context, index) {
+        final member = state.team!.players![index];
+        return MemberCell(member: member);
```

Comment:
can't we use UserDetailCell here with little modification like, passing a flag and show phone-number accordingly??